ansible/roles/schulcloud-client-core/templates/configmap.yml.j2

```diff
@@ -106,8 +106,8 @@ data:
   GLOBAL_ANNOUNCEMENT_ADMIN_TEACHER: "{{ GLOBAL_ANNOUNCEMENT_ADMIN_TEACHER|default("", true) }}"
 {% endif %}
   FEATURE_ALERTS_ENABLED: "{{ FEATURE_ALERTS_ENABLED|default("false") }}"
-  ALERT_STATUS_URL: "{{ ALERT_STATUS_URL|default("https://status.hpi-schul-cloud.de", true) }}"
-  ALERT_STATUS_API_URL: "{{ ALERT_STATUS_URL|default("https://status.hpi-schul-cloud.de", true) }}/api/v1"
+  ALERT_STATUS_URL: "{{ ALERT_STATUS_URL|default("https://status.dbildungscloud.de", true) }}"
+  ALERT_STATUS_API_URL: "{{ ALERT_STATUS_URL|default("https://status.dbildungscloud.de", true) }}/api/v1"
```

Comment:
the api url might not be in use from the client...

Reply:
So it is not necessary or why was it there in the first place?

Reply:
If not necessary and nobody contradicts I would remove the API URL from the configmap in legacy client

Reply:
if you cant find a mention of it in the client code, get rid of it.